hoopl.cabal

```diff
@@ -24,6 +24,12 @@ Source-repository head
   Type:       git
   Location:   http://git.haskell.org/packages/hoopl.git
 
+flag testcoverage {
```

Comment:
Why do you use `{}`-syntax here? Using that may cause problems for cabal parsers supporting only the "simple" grammar